packages/snap-preact/components/src/components/Organisms/Facet/Facet.tsx

```diff
@@ -384,6 +384,7 @@ export const Facet = observer((properties: FacetProps): JSX.Element => {
 										name={'reset-facet'}
 										onClick={() => {
 											facet?.clear.url.link.onClick();
+											facet.toggleCollapse();
```

Comment:
🤔 ... so just end up calling this twice to "undo" the first one?

Would it be better to just prevent the click from bubbling up to the parent "header" event?